### ISSUE TITLE
PVT for Faster RCNN

### ISSUE THREAD

**Issue by chandlerbing65nm** [whai362/PVT#81]
Can I try your PVT and PVTv2 to Faster-RCNN? Will it also give a big improvement (if you tried simulating this)?

**Comment by lingblessing** [comment]
Hi, have you tried it?

**Comment by chandlerbing65nm** [comment]
> Hi, have you tried it?

No, I have not tried it. I wonder how much will be the improvement. In Faster-RCNN, RPN is an attention-based network, that's why I wonder about the performance since both the RPN and PVT are attention-based.
